sqlflow/client.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import sys
+import logging
+
+import grpc
+import sqlflow_pb2
+import sqlflow_pb2_grpc
+import google.protobuf.wrappers_pb2 as wrapper
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--url", type=str, help="server url", action="store", required=True)
+
+#TODO(tonyyang-svail): move logging config elsewhere
+logging.basicConfig(format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
```

Comment:
may we create variable for logger? 
_LOGGER = logging.getLogger(__name__)

By the way, for logging config, just keep default and let user set it.

Reply:
Done.